README.md

```diff
@@ -27,15 +27,10 @@ This package provides a KeyShot plugin script that creates jobs for AWS Deadline
 
 1. Install deadline and Pyside6
```

Comment:
Should we add a note here that deadline should be > 0.48.7 for sticky settings to work at least? The update to the pyproject.toml won't enforce anything for the submitter, only the adaptor, so it's up to the user to have a new enough version of deadline present for sticky settings to work.

Reply:
Added a note and updated the recommended `pip install` instruction!